src/App.js

```diff
@@ -7,19 +7,19 @@ import './App.scss';
 
 class App extends Component {
 
-    componentDidMount () {
-        insights.chrome.init();
-        insights.chrome.identifyApp('dashboard');
-        insights.chrome.navigation(buildNavigation());
+    // componentDidMount () {
+    //     insights.chrome.init();
+    //     insights.chrome.identifyApp('dashboard');
+    //     insights.chrome.navigation(buildNavigation());
 
-        this.appNav = insights.chrome.on('APP_NAVIGATION', event => this.props.history.push(`/${event.navId}`));
-        this.buildNav = this.props.history.listen(() => insights.chrome.navigation(buildNavigation()));
-    }
+    //     this.appNav = insights.chrome.on('APP_NAVIGATION', event => this.props.history.push(`/${event.navId}`));
+    //     this.buildNav = this.props.history.listen(() => insights.chrome.navigation(buildNavigation()));
+    // }
 
-    componentWillUnmount () {
-        this.appNav();
-        this.buildNav();
-    }
+    // componentWillUnmount () {
+    //     this.appNav();
+    //     this.buildNav();
+    // }
 
```

Comment:
This code is required in order to register nav links with the dashboard App so that we can route to the correct page based on the nav link selected.